src/components/ImageGrid/LoadMoreButton.vue

```diff
@@ -31,10 +31,14 @@ export default {
       type: Boolean,
       default: false,
     },
+    mediaType: {
+      type: String,
+      default: 'image',
+    },
   },
   computed: {
     finishedLabel() {
-      const type = this.$t('browse-page.search-form.image')
+      const type = this.$t(`browse-page.search-form.${this.mediaType}`)
```

Comment:
Interpolation like this will break our source url detection code for the `.pot` file, correct? I think it's okay if a few translations are missing the reference to the source code. Alternatively this would have to be written as a switch or if/else block with each media type:  `this.$t(`browse-page.search-form.image`)`, `this.$t(`browse-page.search-form.audio`)`, and so on.

Reply:
I haven't thought about it :)